lib/client-assets.php

```diff
@@ -495,25 +497,21 @@ function gutenberg_register_vendor_scripts() {
 		'https://unpkg.com/moment@2.22.1/' . $moment_script,
 		array()
 	);
-	$tinymce_version = '4.7.2';
-	gutenberg_register_vendor_script(
-		'tinymce-latest',
-		'https://unpkg.com/tinymce@' . $tinymce_version . '/tinymce' . $suffix . '.js'
-	);
+	$tinymce_version = '4.7.11';
 	gutenberg_register_vendor_script(
 		'tinymce-latest-lists',
 		'https://unpkg.com/tinymce@' . $tinymce_version . '/plugins/lists/plugin' . $suffix . '.js',
```

Comment:
We could load these from core too, except the table plugin. Maybe better like this though.

Reply:
I thought they were loaded automatically so I removed this but it didn't work. I believe we could use the Core provided once but it's not as urgent as TinyMCE.